src/main/java/com/study/realworld/article/domain/ArticleContent.java

```diff
@@ -25,7 +26,7 @@
     @JoinTable(name = "article_tag",
         joinColumns = @JoinColumn(name = "article_id"),
         inverseJoinColumns = @JoinColumn(name = "tag_id"))
-    @ManyToMany(cascade = CascadeType.PERSIST)
+    @ManyToMany(cascade = CascadeType.PERSIST, fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany( fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
```
별거 아닐 수 있는데 순서는 중요한 쪽으로 놓는게 좋을 것 같아요

Reply:
안그래도 순서에 대해서 계속 고민하고 있었는데 확인 감사합니다 :)